addons/google_calendar/models/google_credentials.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+import requests
+from odoo.addons.google_calendar.models.google_sync import google_calendar_token
+from datetime import timedelta
```

Comment:
I'm surprised the linter doesn't complain about those imports @Xavier-Do has that been disabled?